### PR TITLE
Add skeleton UI

### PR DESCRIPTION
**Pull request by horizon0708**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -6,11 +6,9 @@ import { BrowserRouter } from 'react-router-dom';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
-  <React.StrictMode>
-    <BrowserRouter>
-      <App />
-    </BrowserRouter>
-  </React.StrictMode>,
```
Comment (horizon0708) [inline src/index.js:13]: @behrouz2115 This is what was causing useEffect to run twice in React 18. So you don't need to do `ref.hasRun` anymore. I was wrong 🙏

Comment (behrouz2115) [reply src/index.js:13]: thank you James it was good point.

**Review by behrouz2115 — COMMENTED** [review]
I checked all files. Now can I pull it?

**Review by behrouz2115 — CHANGES_REQUESTED** [review]
can I merge it or do I need your permission?

**Comment by horizon0708** [comment]
You've requested changes instead of approving though 😓

**Comment by horizon0708** [comment]
When you request change, it means you want me to fix something btw
image
image

**Review by behrouz2115 — APPROVED** [review]

**Pull request merged**